test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/BuildIntegrationTest.cs

```diff
@@ -630,6 +630,28 @@ public async Task Building_NETCoreApp21TargetingProject()
             Assert.FileExists(result, OutputPath, "SimpleMvc21.Views.pdb");
         }
 
+        [Fact]
+        [InitializeTestProject("SimpleMvc21")]
+        public async Task Building_WorksWhenMultipleRazorConfigurationsArePresent()
+        {
+            TargetFramework = "netcoreapp2.1";
+            AddProjectFileContent(@"
+<ItemGroup>
+    <RazorConfiguration Include=""MVC-2.1"">
+      <Extensions>MVC-2.1;$(CustomRazorExtension)</Extensions>
```

Comment:
Where is `CustomRazorExtension` set?

Reply:
Copied this from our targets: https://github.com/aspnet/Razor/blob/master/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Configuration.targets#L68-L70

Reply:
Ah I see. Looks like it's there for extensibility